kernel/build/config-amd64

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 6.1.46 Kernel Configuration
+# Linux/x86_64 6.1.51 Kernel Configuration
```

Comment:
this changed from `x86` to `x86_64` since `CARCH` is set properly in the `kernl/build/pkg.yaml`